lib/rubocop/cop/performance/count.rb

```diff
@@ -61,6 +78,14 @@ def autocorrect(node)
 
         private
 
+        def should_run?
+          if config[RAILS] && config[RAILS][ENABLED]
```

Comment:
I had initially forgotten about the command line flags. It turns out that I will not have to do anything special to handle the command line flags. `Runner#inspect_file` calls `enable_rails_cops` if one of the command line flags is passed in and sets the `config['Rails']['Enabled']` flag.


```ruby
def inspect_file(processed_source)
  config = @config_store.for(processed_source.path)
  enable_rails_cops(config) if @options[:rails]
  team = Cop::Team.new(mobilized_cop_classes(config), config, @options)
  offenses = team.inspect_file(processed_source)
  @errors.concat(team.errors)
  @warnings.concat(team.warnings)
  [offenses, team.updated_source_file?]
end

def enable_rails_cops(config)
  config['Rails'] ||= {}
  config['Rails']['Enabled'] = true
end
```